App/Containers/OnboardingScreen/index.tsx

```diff
@@ -166,8 +162,8 @@ class OnboardingScreen extends React.Component<Props, State> {
             style={[CONTAINER]}
             containerStyle={{ marginBottom: spacing._016 }}
             indicatorColor={color.accent2_2}
-            startPage={this.state.currentPage}
```

Comment:
This is actually the thing I'm most suspicious could have caused problems. This `render` method gets called over and over and over again. `startPage` is, in theory, something that should be static, but we were setting it to `state.currentPage` which is changing as the user progresses through the onboarding. You would think that the pager component should deal with this correctly, but there's a good change it doesn't and may be causing our bug.

Reply:
So, omitting this prop, it defaults to `0`, and we never pass in any other value.